src/components/Card/Card.tsx

```diff
@@ -1,16 +1,47 @@
 import React from 'react';
 import { Post } from '../App/App';
+import 'dayjs';
 import './Card.css';
+import dayjs from 'dayjs';
 
 interface CardProps {
   post: Post;
 }
 
 export const Card: React.FC<CardProps> = ({ post }) => {
+  const formatDateCreated = () => {
+    let now = dayjs();
```

Comment:
I was under the impression this was something BE created serverside?

Reply:
Yea, we will add the created date when this is added to our DB, so I don't think you'll need to store it (if I'm understanding what @k-atwhite is saying here. We will also update the "updated_at" date every time we update the resource.

Reply:
This is just to format the createdAt that is coming from the BE, so that if within 24 hours it will say "2hours ago" if farther in the past, it will say the full date on the DOM.